src/heartbeat/index.js

```diff
@@ -0,0 +1,20 @@
+const { logNode } = require('../utils/debug');
+const pulse = require('./pulse');
+
+async function loop(bridgeState, sender) {
+  try {
+    await pulse(bridgeState, sender);
+    setTimeout(() => loop(bridgeState, sender), 60000);
```

Comment:
I'd like to move this const `60000` somewhere in the configuration but I'm not sure where I should put it.

Reply:
I've added `./defaults`

Reply:
bridgeState is not the best place for these IMO. I'd put it in `cliArgs` with defaults.